.github/workflows/acceptance-tests.yml

```diff
@@ -40,35 +40,58 @@ jobs:
           git fetch --prune --unshallow
           echo "::set-output name=TAG_NAME::$(git describe --tags $(git rev-list --tags --max-count=1))"
 
-  wait-for-oclif-releases:
-    name: Wait for oclif platform executables Release to complete
-    runs-on: ubuntu-latest
+  wait-for-brew-releases:
+    name: Wait for brew Release to complete
+    runs-on: macos-11
     continue-on-error: true
     needs: [ get-branch, get-tag ]
+    strategy:
+      matrix:
+        node-version: [ 16.x, 14.x, 10.x ]
     outputs:
-      FAILURE_TAG_OCLIF: ${{ steps.failure_oclif.outputs.FAILURE_TAG_OCLIF }}
+      FAILURE_TAG_BREW: ${{ steps.failure_brew.outputs.FAILURE_TAG_BREW }}
     steps:
       - name: Checkout cli repo
         uses: actions/checkout@v2
-      - name: Wait for Oclif Release
+      - name: Wait for oclif executables Release
         run: source .github/scripts/trigger-and-wait.sh
         env:
           INPUT_WORKFLOW_FILE_NAME: oclif-release.yml
           INPUT_REF: ${{ needs.get-branch.outputs.branch }}
-      - name: Perform install and test on binaries
-        id: oclif_acceptance_run
+      - name: Global - Testing installation of Twilio CLI (1)
+        if: ${{ github.respository != 'twilio/draft-release-branch' }}
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          REPO_NAME: ${{ github.repository_owner }}
         run: |
-          gh release download ${{ needs.get-tag.outputs.TAG_NAME }} --pattern '*.deb'
-          sudo dpkg -i twilio-${{ needs.get-tag.outputs.TAG_NAME }}.deb
-          ls
-          twilio --version
-          twilio api --help
-      - name: setup oclif failure output
+          brew tap $REPO_NAME/brew && brew install twilio
+          node test/asserts/validate_tests.js
```

Comment:
`.github/scripts/cmds-to-test.sh` same file should be used to test on `main` branch too. As you pointed corrctly, it would be confusing if test file doesn't exist and is being used below (line#81). This [PR](https://github.com/twilio/twilio-cli/pull/358)  has details about the usage of this test file.

Reply:
Changes made